qboost/tests/test_demo.py

```diff
@@ -1,29 +1,15 @@
-import unittest
 import os
-import sys
-
-from dwave.system.samplers import DWaveSampler
-
-try:
-    DWaveSampler()
-    _config_found = True
-except ValueError:
-    _config_found = False
+import subprocess
+import unittest
 
-# /tests/integration/test_demo.py
-PY2 = sys.version_info.major == 2
+# /path/to/demos/qboost/tests/test_demo.py
 project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 
-@unittest.skipUnless(_config_found, "no configuration found for accessing the D-Wave System")
 class TestDemo(unittest.TestCase):
     def test_smoke(self):
         """run demo.py and check that nothing crashes"""
 
         demo_file = os.path.join(project_dir, 'demo.py')
+        subprocess.check_output(["python", demo_file, "--mnist", "--wisc"])
```

Comment:
Looks like you should actually check the exit status here, not just run it. At least, judging by the old code...

Reply:
I thought I wouldn't need to check because `subprocess.check_output(..)` automatically throws an exception for non-zero exits. https://docs.python.org/3/library/subprocess.html#subprocess.check_output

Reply:
I forgot about that. Excellent than!